src/Vendoring/README.md

```diff
@@ -10,31 +10,39 @@ git checkout tags/Instrumentation.SqlClient-1.7.0-beta.1
 
 ### Instructions
 
-- Copy required files from `src/Shared`:
+- Copy files from `src/OpenTelemetry.Instrumentation.SqlClient` to `src/Vendoring/OpenTelemetry.Instrumentation.SqlClient`:
+    - `**\*.cs` minus `AssemblyInfo.cs`
+- Update `SqlActivitySourceHelper` with:
+  ```csharp
+  public const string ActivitySourceName = "OpenTelemetry.Instrumentation.SqlClient";
+  public static readonly Version Version = new Version(1, 7, 0, 1173);
+  ```
+- Copy files from `src/Shared` to `src/Vendoring/OpenTelemetry.Instrumentation.SqlClient/Shared`:
     - `DiagnosticSourceInstrumentation\*.cs`
     - `ExceptionExtensions.cs`
     - `Guard.cs`
     - `SemanticConventions.cs`
 
-## OpenTelemetry.Instrumentation.SqlClient
+## OpenTelemetry.Instrumentation.StackExchangeRedis
 
 ```console
-git clone https://github.com/open-telemetry/opentelemetry-dotnet.git
+git clone https://github.com/open-telemetry/opentelemetry-dotnet-contrib.git
 git fetch --tags
-git checkout tags/Instrumentation.SqlClient-1.7.0-beta.1
+git checkout tags/Instrumentation.StackExchangeRedis-1.0.0-rc9.13
```

Comment:
What's your thoughts on putting these versions in? As a way to document which version we are currently on? How do we plan on keeping it up to date?

Reply:
Just to show which versions are used right now. When there is a new release this has to be updated. Otherwise there is no way to know which is currently used and if work needs to be done. Also the way the otel repos is done each lib is release individually which doesn't help.